test/integration/fixtures/build-withConfig/tsdx.config.js

```diff
@@ -3,6 +3,10 @@ const autoprefixer = require('autoprefixer');
 const cssnano = require('cssnano');
```

Comment:
Can't comment at the top of the file, but per my other comment, we probably want to add `// @ts-check` to the top of this and the example file.

Adding `allowJs` + `checkJs` to templates' and the tests' `tsconfig` probably makes more sense long-term, but there will probably be some ramifications to that and may want to wait to do that until #871 is fixed and creates two `tsconfig`s, one for `build` and one for type-checking. I definitely need to test that more before adding.